GrandCentralBoard/Widgets/Harvest/BillableDates.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  BillableDates.swift
+//  GrandCentralBoard
+//
+//  Created by Maciek Grzybowski on 03.06.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Foundation
+
+
+final class BillableDates {
```

Comment:
In `BillableDates`, I'm using `NSCalendar`. `NSDate` is just a wrapper over raw timestamp and shouldn't know about the `NSCalendar`. Thus, I don't want do introduce it in extension 👎 
```swift
let _ = NSDate.arrayOfPreviousDays(5, referenceDate: today, includeWeekends: false, calendar: NSCalendar)
```